ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml

```diff
@@ -132,6 +132,86 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws-multitenant,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-aws-ovn-kubernetes
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cluster-network-operator-master-e2e-aws-ovn-kubernetes
+    rerun_command: /test e2e-aws-ovn-kubernetes
+    run_if_changed: ([^d]|[^m]d|[^.]md)$
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-aws-ovn-kubernetes-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-aws-ovn-kubernetes
+        - --template=/usr/local/e2e-aws-ovn-kubernetes
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_NETWORK_MANIFEST
+          value: |
+            spec:
+              defaultNetwork:
+                type: OVNKubernetes
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          value: |
+            resources:
+              '*':
+                limits:
+                  memory: 4Gi
+                requests:
+                  cpu: 100m
+                  memory: 200Mi
+            tests:
```

Comment:
you don't need this

Reply:
it yells at me about resources needing to be defined until I do in the rehearse jobs

Reply:
I mean the tests stanza, not resources